Banking Application Example for Introduction (#162)
ietf-teep/architecture: draft-ietf-teep-architecture.md

those application components perform security sensitive operations or operate on
sensitive data. An application component running inside a TEE is referred to as a
Trusted Application (TA), while an application running outside any TEE
is referred to as an Untrusted Application. In the example of a banking application, 
code that relates to the authentication protocol could reside in a TA while the 

> application logic including HTTP protocol parsing would be contained in the 

Reviewer: ```suggestion
application logic including HTTP protocol parsing could be contained in the 
```

> Untrusted Application. The precise code split is ultimately a decision of the 

Reviewer: ```suggestion
Untrusted Application.  In addition, processing of credit card numbers or account balances could be done in a TA as it is sensitive data.
The precise code split is ultimately a decision of the 
```

> developer based on the asssets he or she tries to protected according to the thread model. 

Reviewer: ```suggestion
developer based on the assets he or she wants to protect according to the threat model. 
```
